modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Checkbox/Checkbox.es.js

```diff
@@ -63,11 +63,12 @@ const Switcher = ({
 							{showLabel && label}
 
 							{required && (
-								<ClayIcon
-									className="reference-mark"
-									spritemap={spritemap}
-									symbol="asterisk"
-								/>
+								<span className="ddm-label-required reference-mark">
```

Comment:
Can these classes be applied directly on the `ClayIcon`? If not use a `<div />` instead of `<span />` improve HTML semantics

Reply:
I tried uses in ClayIcon, but not worked correctly. I used as base the `ReactFieldBase.es.js` for to do this changes.

Reply:
With `<span />`
![Screenshot from 2021-08-12 11-01-43](https://user-images.githubusercontent.com/38770741/129211101-3d7fc58f-82e3-481b-a6f0-378e9b4c896c.png)
![Screenshot from 2021-08-12 11-01-33](https://user-images.githubusercontent.com/38770741/129211109-8f8042e7-31d0-4d68-9d48-9c7b5a1ff416.png)

Reply:
Without:
![Screenshot from 2021-08-12 11-05-08](https://user-images.githubusercontent.com/38770741/129211240-ab9be082-33e9-44df-b723-e05fab4878d3.png)
![Screenshot from 2021-08-12 11-04-56](https://user-images.githubusercontent.com/38770741/129211244-b7778063-c8db-444b-b824-eb7ac1516e47.png)